build/README.md

```diff
@@ -0,0 +1,15 @@
+The `build` module is used as an easy way to obtain a WildFly distribution containing the feature pack.
+Resulting files from this module:
+
+- `wildfly-{WILDFLY_VERSION}-mp-graphql-{FEATUREPACK_VERSION}`
+  - Thin (maven-based) distribution of WildFly with the feature pack installed
+- `wildfly-{WILDFLY_VERSION}-mp-graphql-{FEATUREPACK_VERSION}-full`
+  - Full (including all JARs) distribution of WildFly with the feature pack installed
+- `wildfly-{WILDFLY_VERSION}-mp-graphql-{FEATUREPACK_VERSION}.zip`
+  - Zip with the thin (maven-based) distribution of WildFly with the feature pack installed 
+- `wildfly-{WILDFLY_VERSION}-mp-graphql-{FEATUREPACK_VERSION}-full.zip`
+  - Zip with the full distribution of WildFly with the feature pack installed
+- `wildfly-{WILDFLY_VERSION}-mp-graphql-{FEATUREPACK_VERSION}-overlay.zip`
```

Comment:
I create this only from the "full" distribution because it does not seem to make a lot of sense to do with the thin Maven-based dist, but I might be convinced otherwise, @t1